muninlite.conf

```
@@ -1,3 +1,6 @@
 # the following variables are added to the top of the assembled muninlite script
 NTP_PEER="pool.ntp.org"
 DF_IGNORE_FILESYSTEM_REGEX="(none|unknown|rootfs|iso9660|squashfs|udf|romfs|ramfs|debugfs|cgroup_root|devtmpfs)"
+
+#INTERFACE_NAMES_OVERRIDE="eth0"
+#INTERFACE_NAMES_OVERRIDE="$(sed 's/^ *//; s/:.*$//; / /d' /proc/net/dev | grep -P '^(eth|wlan|en)')"
```

What about `grep -E` to make this more portable? I guess not a big deal since it's just a commented example.

```suggestion
#INTERFACE_NAMES_OVERRIDE="$(sed 's/^ *//; s/:.*$//; / /d' /proc/net/dev | grep -E '^(eth|wlan|en)')"
```